docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/snowflake-loader/index.md

```diff
@@ -72,7 +81,11 @@ The loader takes two configuration files:
 - a `config.hocon` file with application settings
 - an `iglu_resolver.json` file with the resolver configuration for your [Iglu](https://github.com/snowplow/iglu) schema registry.
 
-An example of the minimal required config for the Snowflake loader can be found [here](https://github.com/snowplow/snowplow-rdb-loader/blob/master/config/snowflake.config.minimal.hocon) and a more detailed one [here](https://github.com/snowplow/snowplow-rdb-loader/blob/master/config/snowflake.config.reference.hocon). For details about each setting, see the [configuration reference](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/rdb-loader-configuration-reference/index.md).
+An example of the minimal required config for the Snowflake loader on AWS can be found [here](https://github.com/snowplow/snowplow-rdb-loader/blob/master/config/loader/aws/snowflake.config.minimal.hocon) and a more detailed one [here](https://github.com/snowplow/snowplow-rdb-loader/blob/master/config/loader/aws/snowflake.config.reference.hocon). 
+
+An example of the minimal required config for the Snowflake loader on GCP can be found [here](https://github.com/snowplow/snowplow-rdb-loader/blob/master/config/loader/gcp/snowflake.config.minimal.hocon) and a more detailed one [here](https://github.com/snowplow/snowplow-rdb-loader/blob/master/config/loader/gcp/snowflake.config.reference.hocon). 
```

Comment:
This might be nice as a little table?

| Cloud |        Minimal Configuration        |         Extended Configuration        |
|:-----:|:-----------------------------------:|:-------------------------------------:|
| AWS   | /aws/snowflake.config.minimal.hocon | /aws/snowflake.config.reference.hocon |
| GCP   | /gcp/snowflake.config.minimal.hocon | /gcp/snowflake.config.reference.hocon |